plugins/ssrmatching/ssrmatching.ml4

```diff
@@ -12,8 +12,7 @@
  * we thus save the lexer to restore it at the end of the file *)
 let frozen_lexer = CLexer.get_keyword_state () ;;
 
-(*i camlp4use: "pa_extend.cmo" i*)
-(*i camlp4deps: "grammar/grammar.cma" i*)
+(*i camlp5use: "pa_extend.cmo" i*)
```

Comment:
This directive was also discontinued according to `dev/doc/build-system.txt`, so just remove the line.